website/docs/cli/config.mdx

```diff
@@ -0,0 +1,226 @@
+﻿---
+title: Config File
+description: Overview and example usages of the generate license file CLI config file
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+The generate-license-file CLI can be configured using a config file so that you don't need to re-enter the same
+config options every time. The config file also allows you to specify additional more complex options that are not
```

Comment:
nit: Not sure if this is prettier/just GH PR-viewer/required for Docusaurus but we should rely on editors for text wrapping rather than manually
inserting new-lines. Otherwise we end up with cut sentences on small viewports.

Reply:
I'm not sure if we see the same thing on the GH website or not but here's how I wrote it in my IDE.

<img width="817" alt="image" src="https://github.com/TobyAndToby/generate-license-file/assets/22356255/2a4075d5-bdf5-40ce-911d-2c88dfe9c2ca">

When adding new lines I've been trying to keep them under/around the 120 char limit (You can see my IDE renders a vertical bar there) otherwise paragraphs will run horizontally for what feels like forever.

I'm also not sure your concern is the case - markdown ignores single line breaks by themselves, so you're safe to insert them wherever you want. You can see here that there's no artifacts of the return lines from the markdown present in the HTML or the rendered page.

<img width="1035" alt="image" src="https://github.com/TobyAndToby/generate-license-file/assets/22356255/86ebd6de-0578-4a13-9aa0-e1ada9bd9348">

In markdown you need to leave two spaces at the end of a line in order to force a return in the rendered output.

<img width="818" alt="image" src="https://github.com/TobyAndToby/generate-license-file/assets/22356255/54798121-7f34-4092-a294-866e5c5f02eb">

Which is why our editor config has `trim_trailing_whitespace` turned off for md files (though I've just noticed that it should do it for mdx files as well, so I'll push that change up).

<img width="219" alt="image" src="https://github.com/TobyAndToby/generate-license-file/assets/22356255/5ee71d49-e84c-406e-bf98-35bc77ad3f7d">

Reply:
Ah interesting, [taken from the Prettier docs] it seems GitHub have an unusual approach to markdown, which I guess is what you're probably used to?.

This shouldn't affect us though?

<img width="607" alt="image" src="https://github.com/TobyAndToby/generate-license-file/assets/22356255/0b3010cf-8eef-46a2-81a3-feac39fcce86">